### PR TITLE
Fixed PK encoding unused during import

### PR DESCRIPTION
**Pull request by alexeykudinkin**
Typo?

### CODE REVIEW

**Comment by indutny** [comment]
Maybe... But why are the tests failing?

File: lib/elliptic/ec/key.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ KeyPair.prototype.getPrivate = function getPrivate(enc) {
 };
 
 KeyPair.prototype._importPrivate = function _importPrivate(key, enc) {
-  this.priv = new bn(key, 16);
+  this.priv = new bn(key, enc);
```
Comment (indutny) [inline lib/elliptic/ec/key.js]: May be `enc || 16`?

**Comment by alexeykudinkin** [comment]
Yeah, you're right. 
Sorry for not firing tests before PR.

**Comment by indutny** [comment]
oh, forgot to say. Landed, thank you!

**Pull request closed without merging**